src/gamma_inc.jl

```diff
@@ -142,7 +141,251 @@ function rgammax(a::Float64,x::Float64)
             return (a*exp(t))*(1.0 + rgamma1pm1(a))
         end
     end
+end
+"""
+    auxgam(x)
+
+Compute function `g` in ``1/\\Gamma(x+1) = 1+x*(x-1)*g(x)``, -1 <= x <= 1 
+"""
+function auxgam(x::Float64)
+    dr = zeros(18)
+    if x < 0
+        return -(1.0+(1+x)*(1+x)*auxgam(1+x))/(1.0-x)
+    else
+        dr[1] = -1.013609258009865776949
+        dr[2] = 0.784903531024782283535e-1
+        dr[3] = -0.12790434869623468120e-2
+        dr[4] = 0.67588668743258315530e-2
+        dr[5] = 0.462939838642739585e-4
+        dr[6] = 0.43381681744740352e-5
+        dr[7] = -0.5326872422618006e-6
+        dr[8] = 0.172233457410539e-7
+        dr[9] = 0.8300542107118e-9
+        dr[10] = -0.10553994239968e-9
+        dr[11] = 0.39415842851e-11
+        dr[12] = 0.362068537e-13
+        dr[13] = -0.107440229e-13
+        dr[14] = 0.5000413e-15
+        dr[15] = -0.62452e-17
+        dr[16] = -0.5185e-18
+        dr[17] = 0.347e-19
+        dr[18] = -0.9e-21
+        t = 2*x - 1.0
+        return chepolsum(t,dr)
+    end
 end   
+"""
+    loggamma1p(a)
+
+Compute ``log(\\Gamma(1+a))`` for -1 < a <= 1.
+"""
+function loggamma1p(x::Float64)
+    return -log1p(x*(x-1.0)*auxgam(x))
```

Comment:
this doesn't seem to be giving correct values: 
```
julia> SpecialFunctions.loggamma1p(0.5)
-0.1350859041360025

julia> loggamma(1.5)
-0.12078223763524522
```

Would be good to add some test for this.

Reply:
This was due to a coefficient mismatch in `auxgam` function. Fixed now.